### PR TITLE
Expression Evaluater (REPL support with inspector sequential results)

### PR DESCRIPTION
**Pull request by LinqLover**
> **Note**
> This is a re-make of #60134 by @rohanrhu (see https://github.com/godotengine/godot/pull/60134#issuecomment-1535310501). I have no idea what I'm doing, I just ran `git rebase`. I did not compile or try anything out. All glory and expertise go to the original author.

### CODE REVIEW

File: modules/visual_script/visual_script.cpp
Comment (dalexeev) [inline modules/visual_script/visual_script.cpp:1]: Visual Script support [was removed](https://godotengine.org/article/godot-4-will-discontinue-visual-scripting/) in 4.0.

**Comment by MewPurPur** [comment]
After looking at the old issue and its proposal, I still don't really get what this is. Looks like it has a lot of support though...

**Comment by LinqLover** [comment]
> The old issue and its proposal aren't well-written and don't explain anything, although, they have a lot of support... I still don't really get what this is though. Could someone explain?

Well, in a nutshell, it should enable users to evaluate custom queries during debugging, and all evaluation results are displayed in a list. This is a common feature to many delivery, it improves interactivity and explorability, it would be great to see this in Godot, too.

**Comment by rohanrhu** [comment]
Dooooon't wooorry I'm back to the Godot Universe. I'll take care of all of my PRs sooon.

**Comment by KoBeWi** [comment]
Superseded by #97647
Thanks for your contribution nonetheless.

**Pull request closed without merging**